src/landpatterns/packages.stanza

```diff
@@ -470,6 +470,17 @@ public defmulti build-all (
   virt:VirtualLP
   ) -> False
 
+public defn package-build-all-seq (
```

Comment:
qq: what's the `seq` suffix here referring to?

Reply:
Sequence - ie, these functions all get called in a particular order. 

I could replace it with `order` if you think that would be less confusing (ie, not confused with list operator `seq`)

Reply:
Yeah, that's probably preferred. I had assumed it referred to the order in which things were being built, as opposed to the operator, but explicitly dodging the possible ambiguity is better.